test/notifier.test.js

```diff
@@ -1057,6 +1057,38 @@ describe("Notifier._buildPayload()", function() {
     done();
   });
 
+  it("should have the most recent frame last", function(done) {
+    var notifier = new Notifier();
+
+    function first() {
+      second();
+    }
+
+    function second() {
+      last();
+    }
+
+    function last() {
+      nonExistant(); 
+    }
+
+    var err;
+    try {
+      first();
+    } catch (e) {
+      err = e
+    }
+    var payload = notifier._buildPayload(new Date(), 'error', 'frame order test', TK(err));
+    expect(payload.data.body.trace.frames).to.be.an('array');
+
+    var numFrames = payload.data.body.trace.frames.length;
+    expect(payload.data.body.trace.frames[numFrames - 3].method).to.equal('first');
+    expect(payload.data.body.trace.frames[numFrames - 2].method).to.equal('second');
+    expect(payload.data.body.trace.frames[numFrames - 1].method).to.equal('last');
```

Comment:
@coryvirok I've been looking at this and I've opened a PR to be merge in the tracekit replacement branch: https://github.com/rollbar/rollbar.js/pull/82.

I would expect `payload.data.body.trace.frames[0].method` to be `last`. Are you sure that actually is not working correctly?